docs/config/README.md

```diff
@@ -4528,3 +4528,33 @@ detect_extensions = ['pst'] # filters *.pst files
 shell = ['pwsh.exe', '-NoProfile', '-Command']
 use_stdin = false
 ```
+
+## Random
+
+`random` modules randomise their output every time the prompt is displayed.
+
+### Options
+
+| Option        | Default              | Description                          |
+| ------------- | -------------------- | ------------------------------------ |
+| `styles`      | `['>', '$', '%']`    | List of style variables to pick from |
+| `symbols`     | `['bold', 'italic']` | List of symbols to pick from         |
+| `format`      | `'[$symbol]($style)` | The format for the module            |
```

Comment:
There is missing an apostrophe at the end of the default configuration: `'[$symbol]($style)'`